pkg/k8s/volumes/crud.go

```diff
@@ -48,29 +49,33 @@ func List(ctx context.Context, namespace, labels string, c kubernetes.Interface)
 
 // CreateForDev deploys the volume claim for a given development container
 func CreateForDev(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset, devPath string) error {
+	if okteto.IsOktetoCloud() && dev.PersistentVolumeStorageClass() != "" {
```

Comment:
Not sure we want to include this logic in the CLI. If at some point we want to allow it, we could not change it for some specific versions of the CLI. Also, the case we want to "fix" for Cloud could happen in other Okteto instances as it is a documented setting: https://www.okteto.com/docs/enterprise/administration/configuration/#volumes.

Reply:
I was gonna say the same, if it's affecting Cloud it should be affecting other Okteto clusters, so I don't know if this should be fixed on the CLI side. Maybe we should return a proper error on the mutation web hook?

Reply:
okey, i did not understood correctly the solution for this :D thought the error should be thrown by the cli, but should be thrown by webhook? When CREATE, the webhook has also a force to use the standard, so we are not even notifying the user that even though they used "X" at the manifest, we are creating the class "Y" :/

Reply:
@teresaromero why don't we do the same on UPDATE at the webhook level?

Reply:
yes, im on it! :) i was confused this morning - will close this PR and open new on webhook